<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherUI</class>
 <widget class="QMainWindow" name="TeacherUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teacher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="teacherIDLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Įveskite dėstytojo ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="groupIDLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Įveskite grupės ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="studentIDLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Įveskite studento ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subjectIDLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Įveskite paskaitos ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gradeLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Įveskite pažymį</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveGradeButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Išsaugoti pažymį</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteGradeButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Pašalinti pažymį</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Atsijungti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewSubjectsButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Peržiūrėti dėstomus
 dalyukus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewGradesButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Žiūrėti studento
balus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findStudentIDButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rasti studento
 ID</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
